@xen-orchestra/lite/src/components/vm/dashboard/VmDashboardCpuUsageChart.vue

```diff
@@ -0,0 +1,84 @@
+<template>
+  <UiCard>
+    <UiCardTitle>
+      {{ t('cpu-usage') }}
+      <template #description>{{ t('last-week') }}</template>
+    </UiCardTitle>
+    <VtsLoadingHero v-if="loading" type="card" />
+    <VtsErrorNoDataHero v-else-if="error" type="card" />
+    <VtsNoDataHero v-else-if="cpuUsage.length === 0" type="card" />
+    <VtsLinearChart v-else :data="cpuUsage" :max-value :value-formatter />
+  </UiCard>
+</template>
+
+<script lang="ts" setup>
+import { RRD_STEP_FROM_STRING, type VmStats } from '@/libs/xapi-stats.ts'
+import type { LinearChartData, ValueFormatter } from '@core/types/chart.ts'
+import VtsErrorNoDataHero from '@core/components/state-hero/VtsErrorNoDataHero.vue'
+import VtsLoadingHero from '@core/components/state-hero/VtsLoadingHero.vue'
+import VtsNoDataHero from '@core/components/state-hero/VtsNoDataHero.vue'
+import UiCard from '@core/components/ui/card/UiCard.vue'
+import UiCardTitle from '@core/components/ui/card-title/UiCardTitle.vue'
+import { computed, defineAsyncComponent } from 'vue'
+import { useI18n } from 'vue-i18n'
+
+const { data } = defineProps<{
+  data: {
+    stats: VmStats | undefined
+    timestampStart: number
+  }
+  loading: boolean
+  error?: string
+}>()
+
+const VtsLinearChart = defineAsyncComponent(() => import('@core/components/linear-chart/VtsLinearChart.vue'))
+
+const { t, n } = useI18n()
+
+const cpuUsage = computed<LinearChartData>(() => {
+  const stats = data.stats
+
+  const timestampStart = data.timestampStart
```

Comment:
To match the way it's done in the other components:
```suggestion
  const { stats, timestampStart } = data
```